ampd/src/handlers/solana_verify_msg.rs

```diff
@@ -254,11 +233,11 @@ mod test {
             voting_verifier,
             RpcCacheWrapper::new(rpc_client, NonZeroUsize::new(10).unwrap()),
             source_chain,
-            broadcast_client,
             rx,
         );
 
-        handler.handle(&event).await.unwrap();
+        let votes = handler.handle(&event).await.unwrap();
+        assert!(!votes.is_empty());
```

Comment:
We dont need the `BroadcasterClient` mock anymore, as its not a dependency anymore. We just need to assert the returned `Vec` instead. This gets repeated in all tests.